x-pack/legacy/plugins/lens/public/xy_visualization/xy_expression.test.tsx

```diff
@@ -12,15 +12,19 @@ import {
   LineSeries,
   Settings,
   ScaleType,
+  GeometryValue,
+  XYChartSeriesIdentifier,
   SeriesNameFn,
 } from '@elastic/charts';
 import { xyChart, XYChart } from './xy_expression';
 import { LensMultiTable } from '../types';
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount, shallow, ReactWrapper } from 'enzyme';
```

Comment:
Let's use `import { mountWithIntl } from 'test_utils/enzyme_helpers'` instead, since that takes care about our i18n framework. Even if we don't have any call in it right now, this will prevent us from needing to change all calls later, in case we would ever introduce any messages within this component.